CONTRIBUTING.md

```diff
@@ -99,7 +99,6 @@ If you're interested in translating the site itself (all the nouns, verbs, and s
 If your language file doesn't exist yet, you can copy the en.json file and start translating from there.
 
 ### JSON Validation
-**Make sure your JSON validates.** You can use either use [JSONLint](http://jsonlint.com/) online or install it locally with `npm install jsonlint -g`.
```

Comment:
I would prefer if we kept this note in some form. IIRC `make test` does more than just lint stuff (doesn't it do a full build?) and anyway it's a better contributor experience to catch errors locally rather than in CI.

Honestly, it seems okay the way it is.

Reply:
I agree this could be kept, I would reword it to `make lint`. We don't have to explain _how_ it is validated or send people of to a website to copy their code into because we have that as part of the build process they can run themselves. This whole section should probably just be "Validation" or "Test" without mentioning JSON in particular and encourage them to test locally before shipping it off to CI.

Reply:
@alerque definitely agree. FWIW I do see some value in this:

> send people of to a website

being kept, since some people want to casually contribute without setting up a full Node/yarn environment. Maybe "just see what CI says" is good enough for this segment though?

Reply:
@strugee Yes, simple is fine.